### ISSUE TITLE
Problems after update to v0.8.6

### ISSUE THREAD

**Issue by chule143** [deployd/deployd#675]
Hi, when I update deployd from v0.8.5 to 0.8.6, when I try to use the dpd command in my folder i get the next message in the console:

```
/usr/lib/node_modules/deployd/bin/dpd:91
  if (test('-f', 'app.dpd')) {
      ^
ReferenceError: test is not defined
    at start (/usr/lib/node_modules/deployd/bin/dpd:91:7)
    at Object.<anonymous> (/usr/lib/node_modules/deployd/bin/dpd:277:31)
    at Module._compile (module.js:460:26)
    at Object.Module._extensions..js (module.js:478:10)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
    at Function.Module.runMain (module.js:501:10)
    at startup (node.js:129:16)
    at node.js:814:3
```

BTW, My node version is v0.12.7 on Linux Mint 17.2

**Comment by NicolasRitouet** [comment]
Hi,

I found the issue.
I removed `shelljs-global`, because I thought it was not used.
I'll put it back and make a new release.

**Comment by NicolasRitouet** [comment]
I just released a fix with version 0.8.7.
Let me know if you have another issue.

**Comment by chule143** [comment]
Great, now it's working normally.

Thanks :)
